src/main/webapp/WEB-INF/view/activity.jsp

```diff
@@ -0,0 +1,43 @@
+<%@ page import="java.util.List" %>
+<%@ page import="java.util.UUID" %>
+<%@ page import="codeu.model.data.User" %>
+<%@ page import="codeu.model.data.Message" %>
+<%@ page import="codeu.model.data.Conversation" %>
+<%@ page import="codeu.model.data.Activity" %>
+<%@ page import="codeu.model.store.basic.UserStore" %>
+<%@ page import="codeu.model.store.basic.MessageStore" %>
+<%@ page import="codeu.model.store.basic.ConversationStore" %>
+<%@ page import="codeu.model.store.basic.ActivityStore" %>
+
+<% List<Activity> activities = (List<Activity>) request.getAttribute("activities"); %>
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Activity</title>
+    <link rel="stylesheet" href="/css/main.css">
+
+    <style>
+        #activity {
```

Comment:
You could edit the css file instead of leaving it here. I think that i would be best but it's not big deal